data/json/items/containers/containers.json

```
@@ -701,6 +701,33 @@
     ],
     "qualities": [ [ "CONTAIN", 1 ] ]
   },
+  {
+    "id": "bowl_wood_large",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "large wooden bowl" },
+    "looks_like": "bowl_plastic",
+    "description": "A large crude-looking wooden bowl which holds liquid.",
+    "ascii_picture": "bowl_wood",
+    "weight": "48 g",
```

```suggestion
    "weight": "88 g",
```

An increase in weight to go along with the increase in volume. I used the same ratio as the volume increase (so ~83% more weight). 

(If you want to get fancy, the weight increases slower than the volume does, because there is empty internal volume. But quick math like this is perfectly fine.)